bark_core/signatures/util.py

```diff
@@ -49,30 +42,33 @@ def parse(cls, identifier: str) -> "Pubkey":
             pass
         raise
 
-    def _parse_pubkey(self, pubkey: str) -> Union[PGPKey, PKey]:
+    def _parse_pubkey(self, pubkey: bytes) -> Tuple[Union[PGPKey, PKey], str]:
         try:
             key, _ = PGPKey.from_blob(pubkey)
             fingerprint = str(key.fingerprint)
             return key, fingerprint
         except Exception:
             pass
         try:
-            key = PKey(data=pubkey)
+            key = PKey(data=pubkey.decode())
```

Comment:
I think `paramiko` expects the pubkey to be in `bytes`, so we shouldn't run `decode()`? The `pubkey` variable seems to be wrongly typed (`str`) from before.

Reply:
Pushed a commit that resolves this. It also ignores missing imports from `paramiko`.